app/models/like.rb

```diff
@@ -0,0 +1,8 @@
+class Like < ApplicationRecord
+  belongs_to :author, class_name: 'User', foreign_key: :author_id
+  belongs_to :post
+
+  def update_likes_counter
+    post.update(likes_counter: post.likes_counter + 1)
+  end
```

Comment:
- [ ] As per previous reviewer requested, It would be nice if we use one of [Rails active callbacks](https://guides.rubyonrails.org/active_record_callbacks.html), the `after_save` method to call the private method `update_likes_counter`. Let use kindly make the method private

Reply:
@Deepakdanger, I thought after_save registers a callback to be called after a record is saved but in this case, its the like that is saved. The update likes counter method is not called after any record is saved but rather the main item that is added to the database by incrementing the count. I stand to be corrected.

Please can you elaborate more for me and do I need to call the after_save callback?

Reply:
- Here we have a create `Like` record after every `like button` clicked , after that, we have to update the likes counter of the post.

Reply:
Got it. Thanks